docs/docs/learn/about_aztec/what_is_aztec.mdx

```diff
@@ -10,15 +10,15 @@ Aztec is an L2 that brings programmable privacy to Ethereum.
 
 A smart contract on Aztec is a collection of functions, written as ZK-SNARK circuits. These circuits can have different modes of execution:
 
-1. Private Functions -- can read and write private state, read historical public state, consume or send messages to / from Ethereum, and read Ethereum state. They can call other private functions in the same contract, or other contracts, and can call public functions.
-2. Public Functions -- can read and write public state, write private state, consume or send messages to / from Ethereum and read Ethereum state. They can call other public functions on the same or other contracts.
-3. Portal Contracts -- these are contracts on Ethereum that can receive messages from Aztec or send messages to Aztec from Ethereum contracts.
+1. **Private Functions** -- can read and write private state, read historical public state, consume or send messages to / from Ethereum, and read Ethereum state. They can call other private functions in the same contract, or other contracts, and can call public functions.
+2. **Public Functions** -- can read and write public state, write private state, consume or send messages to / from Ethereum and read Ethereum state. They can call other public functions on the same or other contracts.
+3. **Portal Contracts** -- these are contracts on Ethereum that can receive messages from Aztec or send messages to Aztec from Ethereum contracts.
 
 Using these different modes of execution, developers can build applications with user privacy, data privacy and code privacy.
 
-- User privacy - transactions may not reveal information about the sender or the recipient.
-- Data privacy - transactions may not reveal information about the payload of the transaction, e.g., the asset or value being transacted.
-- Code privacy - transactions may not reveal the program logic.
+- ((User privacy** - transactions may not reveal information about the sender or the recipient.
```

Comment:
why (( and **?

Reply:
title makes the points bold for better clarity and readability compared to the previous documentation style.